cs/json.texy

```diff
@@ -53,10 +51,14 @@ try {
 }
 \--
 
-Jak odeslat JSON z presenteru?
-==============================
 
-V metodě `action*()` nebo `render*()` zavoláme `$this->sendJson($payload);`, kde `$payload` obsahuje data, která chceme zakódovat do JSONu a odeslat.
+MVC - Presenter
+===============
+
+Jak odeslat JSON z presenteru? V `Presenter::action<>()` metodě zavoláme `$this->sendJson($payload)`, kde `$payload` jsou data, které chceme odeslat v JSON formátu.
```

Comment:
Vim, ze z hlediska volani presenteru je to fuk, ale pro novacky je to mozna az moc "volne", nemeli by data spis pripravovat v action metode a potom tedy i odesilat v action metode? Jenom teoreticky.. :-) Radeji bych jim nabizel jednu volbu nez vicero. Zkuseni si to rozhodnou sami.